autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/LoadAlertConfigurationResponse.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.periscope.api.model;
```

Comment:
LoadAlertConfigurationRequest and LoadAlertConfigurationResponse are identical. Should be a single base class from which they inherit. Alternately, can be replaced by a 'LoadAlertConfiguration'

Reply:
same as above. have kept in sync with existing hierarchy we can refactor all request\responses\converters in a later PR if required.